externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SparkConsoleProgressBar.scala

```diff
@@ -86,7 +86,7 @@ class SparkConsoleProgressBar(
       val w = width - header.length - tailer.length
       val bar =
         if (w > 0) {
-          val percent = w * s.numCompleteTasks / total
+          val percent = w * s.numCompleteTasks.get() / total
```

Comment:
nit: eliminate `()` on pure function invocation without side-effect
```suggestion
          val percent = w * s.numCompleteTasks.get / total
```

Reply:
OK, thx.